my-first-jina-app/README.md

```diff
@@ -308,6 +293,16 @@ Flow@133216[S]:flow is closed and all resources should be released already, curr
 
 You'll know indexing is complete. This may take a little while the first time, since Jina needs to download the language model and tokenizer to deal with the data. You can think of these as the brains behind the neural network that powers the search.
 
+#### Index More Documents
+
+To speed things along, by default Jina is set to index a maximum of 500 [Documents](https://github.com/jina-ai/jina/tree/master/docs/chapters/101#document--chunk). This is great for testing our search engine works, but not so good for searching through every little piece of data.
+
+Once we've verified everything works, we can set it to `50000` (or any number less than `62605`) to index many more lines of the dataset:
+
+```sh
+export $MAX_DOCS=50000
```

Comment:
isn't it `export MAX_DOCS=50000`?

Reply:
On line 8 of `app.py` that is created by cookiecutter, there's the line:

```python
num_docs = os.environ.get('MAX_DOCS', 500)
```

So it's already set to 500. In my original tutorial I set it to 500 for speed. But now that that's the default I put a note about increasing it.

Here's the file in cookiecutter repo: https://github.com/jina-ai/cookiecutter-jina/blob/master/%7B%7B%20cookiecutter.project_slug%20%7D%7D/app.py